providers/implementations/asymciphers/sm2_enc.c

```diff
@@ -7,7 +7,6 @@
  * https://www.openssl.org/source/license.html
```

Comment:
not sure what the diff is on this file?

Reply:
Only change in this file is to remove the extra blankline. The build issue which manifested here was resolved entirely in the build.info file.